pkg/http/core/applications.go

```diff
@@ -761,8 +805,17 @@ func listServerGroups(c *gin.Context, wg *sync.WaitGroup, sgs chan ServerGroup,
 func makeServerGroupMap(pods *unstructured.UnstructuredList) map[string][]Instance {
 	// Map of server group to instances (pods)
 	serverGroupMap := map[string][]Instance{}
+	items := pods.Items
+	// Sort the pods.
+	sort.Slice(items, func(i, j int) bool {
+		if items[i].GetNamespace() != items[j].GetNamespace() {
```

Comment:
Why are we using `GetNamespace` here instead of just `Namespace`? Like to good above.

Reply:
Because in this case, `items` is of type `unstructured.Unstructured` which does not have a `Namespace` field, but does have a `GetNamespace()` function.